.github/workflows/announce-calls.yml

```diff
@@ -0,0 +1,23 @@
+name: Announce calls
+on:
+  schedule:
+    # 8th to the 14th of each month at 9am.
+    - cron: '0 9 8-14 * *'
```

Comment:
What timezone does this use?

Reply:
Good question. Probably UTC? So it would actually run at 4/5 am Eastern (depending on DST). That's probably OK... ?

Reply:
Confirmed UTC per https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule